workflows/python/sdk/README.md

```diff
@@ -36,17 +37,14 @@ expected_stdout_lines:
   - "Purchase of item is  Completed"
 output_match_mode: substring
 background: true
-timeout_seconds: 30
+timeout_seconds: 240
```

Comment:
Can we drop this down to 120 or so? I don't imagine that this test should take more than a minute or so, and dropping it to 2 minutes should be a significant buffer. If this test does take more than 2min, we may have a different problem on our hands